maker/src/routes.rs

```diff
@@ -124,6 +126,12 @@ pub async fn put_offer_params(
     maker: &State<Maker>,
     _auth: Authenticated,
 ) -> Result<(), HttpApiProblem> {
+    // TODO: temporary until the automation can deal with this
```

Comment:
🙃 Might be better to record a ticket for the TODO that points to the code and explains what is to do.

Reply:
I prefer having both :)

Reply:
done.